components/bl-pagination/README.md

```diff
@@ -60,3 +60,58 @@ The component based on external [pagination](https://mui.com/material-ui/react-p
 @bl-customComponent-pagination-button-normalSize: 32px;
 @bl-customComponent-pagination-button-largeSize: 40px;
 ```
+
+## Usage guide
+
+We will display this data table:
+
+![Data Table](./example-images/data-table.png)
+
+Add Data Grid and Pagination component to the page:
+
+![Add Data Grid and Pagination](./example-images/add-data-grid-and-pagination.png)
+
+### Configure the Data Grid component:
+
+Configure the columns to be displayed:
+
+![Configure Data Grid](./example-images/configure-data-grid-1.png)
+![Configure Data Grid](./example-images/configure-data-grid-2.png)
+
+### After we make Row Data Logic for Data Grid
+
+Make variable “pageSize” where we will save how many rows must be in one page
+
+![Make variable](./example-images/make-variable-page-size.png)
+
+Now we need to calculate how many pages we have. For that, we use the block for getting the count of rows in our table and then divide this count by the count of rows on one page(pageSize). This result we round up for integer number:
+
+![Calculate pages](./example-images/calculate-page-count.png)
+
+After we set this result to “Page Data”:
+
+![Set result](./example-images/set-page-count-to-page-data.png)
+
+Now we load the table's data in “return” and calculate page offset, where “currentPage” we will get from the Pagination event “On Page Change”:
+
```

Comment:
Same here.

Forget for requests in `... Logic` handlers.

You should load pages at `On Page Change request` and store them into some variable in Page Data(for example) that will be used by Data Grid